Assets/Scripts/Utils/SteamManager.cs

```diff
@@ -37,13 +38,20 @@ public class SteamManager : MonoBehaviour
     public List<ulong> PlayerIDs = new();
     public delegate void LobbyEvent();
     public LobbyEvent LobbyPlayerUpdate;
+    public LobbyEvent LobbyListUpdate;
+    private List<CSteamID> lobbies = new();
+    public Dictionary<CSteamID, string> Lobbies = new();
```

Comment:
Ideally the `string` value should be an object with s'more interesting values (and the public access could be the .Values of the dict if the key was part of that dict, and the dict key could be the ulong value that is encapsulated in the CSteamID), but I'll let that slide for now as this menu needs a refactor for player count and so on later.

Reply:
Yeah, should be done as soon as we want any other data thais not just a string name